modules/Administration/PasswordManagerSecurity.tpl

```diff
@@ -0,0 +1,97 @@
+<table width="100%" border="0" cellspacing="0" cellpadding="0">
+
+    <!-- Password Security Settings -->
+    <tr>
+        <th align="left" scope="row" colspan="3"><h4>{$MOD.LBL_PWDSEC_SETS}</h4></th>
+    </tr>
+
+    <!-- Expiry Period
+    <tr>
+        <td width="25%" scope="row" valign="middle">
+            {$MOD.LBL_PWDSEC_EXPIRY}
+            {sugar_help text=$MOD.LBL_PWDSEC_EXPIRY_DESC}
+        </td>
+        <td valign="middle">
+            <input name="passwordsetting_expirydays" id="passwordsetting_expirydays" type="number" value="{$config.passwordsetting.expirydays}">
+            {$MOD.LBL_PWDSEC_DAYS}
+        </td>
+        <td>&nbsp;</td><td>&nbsp;</td>
+    </tr>
+    -->
+
+    <!-- Block After Inactivity --> <!-- TODO: Im not 100% sure, is this a password settings?
```

Comment:
Ideally this should be removed. If you still need this in can you please use the smarty method to comment out code instead of HTML method, so that they user doesn't see this in the browser.